crates/stellar-scaffold-cli/src/commands/generate/contract/mod.rs

```diff
@@ -52,24 +53,33 @@ pub enum Error {
 }
 
 impl Cmd {
-    pub async fn run(&self) -> Result<(), Error> {
+    pub async fn run(
+        &self,
+        global_args: &stellar_cli::commands::global::Args,
+    ) -> Result<(), Error> {
         match (&self.from, self.ls, self.from_wizard) {
-            (Some(example_name), _, _) => self.clone_example(example_name).await,
-            (_, true, _) => self.list_examples().await,
-            (_, _, true) => open_wizard(),
+            (Some(example_name), _, _) => self.clone_example(example_name, global_args).await,
+            (_, true, _) => self.list_examples(global_args).await,
+            (_, _, true) => open_wizard(global_args),
             _ => Err(Error::NoActionSpecified),
         }
     }
 
-    async fn clone_example(&self, example_name: &str) -> Result<(), Error> {
-        eprintln!("🔍 Downloading example '{example_name}'...");
+    async fn clone_example(
+        &self,
+        example_name: &str,
+        global_args: &stellar_cli::commands::global::Args,
+    ) -> Result<(), Error> {
+        let printer = Print::new(global_args.quiet);
```

Comment:
This always annoyed me that it wasn't a method on the global_args to create `Print`.